runtime/src/accounts_db.rs

```diff
@@ -4396,7 +4396,7 @@ impl AccountsDB {
             None::<StorageFinder>,
             None::<Box<dyn Iterator<Item = u64>>>,
             is_cached_store,
-            false,
+            true,
```

Comment:
@sakridge , I think you made this change locally yesterday in discussion with carl and me. Can you please confirm? This test was failing because of this.
test_account_grow

Reply:
I had sent you a [github comment](https://github.com/solana-labs/solana/pull/14880#issuecomment-770334958) on this test.

Reply:
I'll take a look.

Reply:
I think the test is just flaky.

Reply:
Summary: this is a correct change.